scripts/entrypoint.sh

```diff
@@ -12,16 +10,16 @@ if [ $# -eq 0 ] ; then
   sh ./scripts/start_server.sh
 elif [ "${1}" = "test" ] ; then
   echo "Run Tests"
-  make test
+  sh ./test/run_tests.sh
 elif [ "${1}" = "async" ] ; then
   sh ./scripts/run_async.sh
-elif [ "${1}" = "init" ] ; then
-  echo "Initialize module"
 elif [ "${1}" = "bash" ] ; then
-  bash
+  echo "This image only supports sh shells"
```

Comment:
not strictly true right now but will be in an upcoming PR

Reply:
Is the plan to trim down the Dockerfile to one of the base Python ones? Or to keep kbase/sdkbase2:python?

Reply:
Yes, I'll using the alpine/python version.